errbot/botplugin.py

```diff
@@ -266,6 +266,19 @@ def callback_message(self, message: Message) -> None:
         """
         pass
 
+    def callback_mention(self, message: Message, mentioned_people: Sequence[Identifier]) -> None:
+        """
+            Triggered if there are mentioned people in message.
```

Comment:
What you mean?
If you mean from usage from some plugin I tried with:

```
def callback_mention(self, message, people):
        if self._bot.bot_identifier.userid in (p.userid for p in people):
            self.send(message.frm, "Somebody mentioned me!", message_type=message.type)
        else:
            self.send(message.frm, "Somebody mentioned someone!", message_type=message.type)
```

Which maybe is not the best approach...